packages/contracts-ops/contracts/scripts/DeployImplementations.s.sol

```diff
@@ -0,0 +1,171 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+/*//////////////////////////////////////////////////////////////
+                                 IMPORTS
+//////////////////////////////////////////////////////////////*/
+// Core
+import {Home} from "@nomad-xyz/contracts-core/contracts/Home.sol";
+import {Replica} from "@nomad-xyz/contracts-core/contracts/Replica.sol";
+import {GovernanceRouter} from "@nomad-xyz/contracts-core/contracts/governance/GovernanceRouter.sol";
+import {IUpdaterManager} from "@nomad-xyz/contracts-core/contracts/interfaces/IUpdaterManager.sol";
+// Bridge
+import {BridgeRouter, EthereumBridgeRouter} from "@nomad-xyz/contracts-bridge/contracts/BridgeRouter.sol";
+import {BridgeToken} from "@nomad-xyz/contracts-bridge/contracts/BridgeToken.sol";
+import {TokenRegistry} from "@nomad-xyz/contracts-bridge/contracts/TokenRegistry.sol";
+// Ops
+import {Config} from "../Config.sol";
+import {JsonWriter} from "../JsonWriter.sol";
+// Utilities
+import {Script} from "forge-std/Script.sol";
+
+abstract contract DeployImplementationsLogic is Script, Config {
+    /*//////////////////////////////////////////////////////////////
+                          DEPLOYED CONTRACTS
+    //////////////////////////////////////////////////////////////*/
+
+    Home home;
+    Replica replica;
+    GovernanceRouter governanceRouter;
+    // type of BridgeRouter differs by domain,
```

Comment:
because EthBridgeRouter is a child contract, you can store it as a bridge router (until/unless you need to access ethbridgerouter-specific interface methods)

Reply:
https://github.com/nomad-xyz/monorepo/pull/527/commits/427d4b27f13c32836317cd54b5aea29cbcb5f951